README.md

```diff
@@ -100,8 +127,9 @@ people.
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-I would like to thank Microverse for their unique way of teach me how to code, and also Freecodecamp for their practical articles that helped me to
-make this project.
+- Microverse Review Team
+- My Coding Partners
+- Microverse Community in Slack
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
Your licence link works fine👍, however, the license itself is missing information related to your project, as seen in the screenshot below:
<img width="815" alt="image" src="https://user-images.githubusercontent.com/37116322/228053940-75d9098e-1fad-4c9f-83d1-e1459623568c.png">

Please update your licence so that it contains information relevant to your project, such as your name and app type. You can follow this [lesson](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to see how

Reply:
Thank you for taking the time to review my pull request, I did the inquired changes accordingly. Except for the commit messages as I did this project last week with all messages in every step, therefore, I'm just finalizing it in this pull request with few commit messages.